sklearn/feature_extraction/text.py

```diff
@@ -1592,7 +1591,7 @@ class TfidfVectorizer(CountVectorizer):
         in the range [0.7, 1.0) to automatically detect and filter stop
         words based on intra corpus document frequency of terms.
 
-    token_pattern : str
+    token_pattern : str, default=r"(?u)\\b\\w\\w+\\b"
```

Comment:
```suggestion
    token_pattern : str, default=r"(?u)\\\\b\\\\w\\\\w+\\\\b"
```

It doesn't look like a backslash because the font-style is italic.

Reply:
... excuse-me @thomasjpfan ... I can't understand this comment ... both `token_pattern` parameter descriptions look the same to me...

Reply:
The suggestion will work without making the string raw. The reason I am advocating for not making the docstring raw is because I do not think there is a way to do multiline type-strings with raw. For example, the `analyzer` parameter had to be updated to remove the `\` at the end of the line.

Reply:
Ok, thanks for finding the way: but splitting the docstring is unnecessary in this case. :)

Reply:
Honestly, I am torn on this one. `\\\\` is a bit ugly to look at, and raw docstrings can be annoying when we need multi-line types. 

If there was a way to do multi-line types with raw docstrings, I would be in favor of the raw docstrings.

Reply:
I understand. Personally I find the `\\\\` quite unreadable and we **don't need** multiline docstring in this function.
My goal here was just to move forward and help in getting rid of an old issue that nobody else had pick up in eight years.
A number of raw docstrings are in scikit-learn, they are especially helpful when maths are in the docstring and latex-like backslashes are everywhere. I will revert the PR to my first version, I hope you don't mind being a bit pragmatic in this case. Thanks.

Reply:
> I will revert the PR to my first version, I hope you don't mind being a bit pragmatic in this case. Thanks.

I do not mind.